GOFevaluation/utils.py

```diff
@@ -246,25 +360,45 @@ def plot_equiprobable_histogram(data_sample, bin_edges, order=None,
     ax.set_ylim(ylim)
 
     ns = apply_irregular_binning(data_sample, bin_edges, order=order)
+    ns = np.array(ns, dtype=float)
+
+    be = limit_bin_edges(bin_edges, data_sample, order)
+    be_first = be[0]
+    be_second = be[1]
 
     # get colormap and norm for colorbar
     cmap_str = kwargs.get('cmap', 'RdBu_r')
     cmap = mpl.cm.get_cmap(cmap_str)
+
+    if(plot_mode == 'count_density'):
+        ns = density_modifier(ns, be_first, be_second, data_sample)
+        cmap_str = kwargs.get('cmap', 'viridis')
+        cmap = mpl.cm.get_cmap(cmap_str)
+    if(plot_mode == 'num_counts'):
+        cmap_str = kwargs.get('cmap', 'viridis')
+        cmap = mpl.cm.get_cmap(cmap_str)
+
     try:
         kwargs.pop('cmap')
     except KeyError:
         pass
+
     if nevents_expected is None:
         norm = mpl.colors.Normalize(vmin=ns.min(), vmax=ns.max())
     else:
         n_bins = get_n_bins(bin_edges)
-        midpoint = nevents_expected / n_bins
         # max deviation
-        delta = max(midpoint - ns.min(), ns.max() - midpoint)
-        sigma_deviation = delta / np.sqrt(midpoint)
-        norm = mpl.colors.Normalize(vmin=-sigma_deviation,
-                                    vmax=sigma_deviation)
-        ns = (ns - midpoint) / np.sqrt(midpoint)
+
+        if(plot_mode == 'count_density' or plot_mode == 'num_counts'):
+            norm = mpl.colors.Normalize(vmin=np.min(ns),
+                                        vmax=np.max(ns))
+        else:
```

Comment:
Also here: 
`elif plot_mode == 'sigma_deviation'` `else:` and then raise an error if none of the allowed